include/SoftKeyMaskComponent.hpp

```diff
@@ -14,11 +14,31 @@
 
 #include "JuceHeader.h"
 
+class SoftKeyMaskDimensions
+{
+public:
+	SoftKeyMaskDimensions() = default;
+
+	int keyHeight = 60;
+	int keyWidth = 60;
+	int keyCount() const;
+	int rowCount = 6;
+	int columnCount = 1;
+	int height = 480;
+	static const std::uint8_t padding = 10;
+
+	/**
+	 * @brief totalWidth
+	 * @return The total width of the softkeymask (including inner and outer column paddings)
+	 */
+	int totalWidth() const;
```

Comment:
not sure about this one, but throughout the C++ stack we tried to keep functions the top, and end with the variables. Maybe try to do the same here? So keyCount() and totalWidth() first and thereafter the variables.